magento/merchdocs#1117: Expand catalog and products viewed compared topics (#1163)
magento/merchdocs: src/marketing/products-viewed-compared.md

1. Expand ![Expansion selector]({% link images/images/btn-expand.png %}) the **Recently Viewed/Compared Products** section and do the following:

>     ![Catalog configuration - recently viewed/compared products]({% link images/images/config-catalog-catalog-recently-viewed-compared-products1.png %}){: .zoom}

Reviewer: Why are there two images in this PR? If you are just updating an existing image, delete the original and replace it.

Author: Hi @dobooth, thank you for reviewing!
I have deleted both images and replaced them with a new one.
Please, check it :)

    [_Recently Viewed/Compared Products_]({% link configuration/catalog/catalog.md %})

>    - Set **Synchronize widget products with backend storage** to synchronize product widget information such as product id with your current product storage availability in database. In order to reuse this info on different devices.

Reviewer 2: ```suggestion
   - Set **Synchronize widget products with backend storage** to synchronize product widget information, such as product ID, with your current product storage availability in the database and reuse this information on different devices.
```

Author: Done :)

   - Set **Show for Current** to the website, store, or store view where the configuration applies.

   - In the **Default Recently Viewed Products Count** field, enter the number of recently viewed products to appear in the list.

>    - In the **Default Recently Compared Products Count**, enter the number of recently compared products to appear in the list.

Reviewer 2: I n=know you did not update this one, but let's fix while we are here.

```suggestion
   - For **Default Recently Compared Products Count**, enter the number of recently compared products to appear in the list.
```

Author: Done :)

>    - In the **Lifetime of products in Recently Viewed Widget** enter how long the products viewed will be shown in the recently viewed list.

Reviewer 2: The default value for these fields is `1000`... what does that equal? 1000 days, hours, something else?

Author: Hi @leslietilling, thank you for the great question!

According to the code, the default value for widgets is 1000 seconds
**Source code:**
https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Catalog/Model/FrontendStorageConfigurationInterface.php#L20
**Regarding the range:**
There is no limit on the maximum amount of time, but at the same time, there is validation - only numbers and no less than zero.

`],
        ‘validate-greater-than-zero’: [
            function (v) {
                if ($.mage.isEmptyNoTrim(v)) {
                    return true;
                }
                v = $.mage.parseNumber(v);
                return !isNaN(v) && v > 0;
 },
            $.mage.__(‘Please enter a number greater than 0 in this field.’)
        ]`
**Source code:**
https://github.com/magento/magento2/blob/2.4/lib/web/mage/validation.js#L823

For more transparency for customers, I'll add clarification to the files and let you know. :)

Author: @leslietilling,
I have implemented some edits for src/marketing/products-viewed-compared.md  file. Could you please review and approve?

Thanks!

>    - In the **Lifetime of products in Recently Viewed Widget** set any time range in seconds, greater than zero. This field shows, how long the products viewed will be shown in the recently viewed list.

Reviewer 2: ```suggestion
   - For **Lifetime of products in Recently Viewed Widget**, enter any time range in seconds, greater than zero. This setting determines how long the viewed products will be shown in the recently viewed list.
```

Author: Done :)

>    - In the **Lifetime of products in Recently Compared Widget** set any time range in seconds, greater than zero. This field shows, how long the products compared will be shown in the recently compared list.

Reviewer 2: ```suggestion
   - For **Lifetime of products in Recently Compared Widget**, enter any time range in seconds, greater than zero. This setting determines how long the compared products will be shown in the recently compared list.
```

Author: Done :)
